submissions/madmaxWMFU/Friends-App/js/app.js

```diff
@@ -0,0 +1,117 @@
+const allFriends = {
+    currentList: [],
+    changeList: [],
+    numberOfFriends: 52,
+    minAge: 0,
+    maxAge: 150
```

Comment:
😱

Reply:
Did I do something wrong? 😔😔

Reply:
I was wondering too.
I think it's just a joke like that form @zonzujiro 
He is amazed by your age policy from 0 t0 150 )))

Reply:
@such7 right :) All ok

Reply:
😃